.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,30 @@
+name: Node.js Package
```

Comment:
This is cool! But I'm guessing it also relies on using Github releases? If so it might be good to document the process for that in the readme as it's not something we really do for any projects at the moment (that I'm aware of)

Reply:
hmm, right, I thought we did actually do version tags on this repo. I've done that for eslint-config-fishbrain-base.

I can switch it to just use master branch. But I think tagging releases is a pretty good practise anyway TBH.

Reply:
Not disagreeing! I think it'd be great to do it, and having the CI run when it happens is a nice way to require it, I just meant it'd be nice to have in the readme a small section saying "User github releases" (or something more useful ;)).

Reply:
npm has the `npm version` command that you can use to tag a new version, both in git and in package.json. It can run `preversion` and  `postversion` scripts to make sure tests pass and push the new  version etc.

Then the github trigger should be
```
on:
  push:
    tags:
      - v*
```

Question is what sort of release process we want for this package though.

Reply:
Do you mean run `npm version` locally or in CI? It'd have to be done locally right? To be honest I don't have a strong feeling about how the release process is done - what we do now is not exactly efficient, so anything will be an improvement I think :)